src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -177,6 +254,54 @@ public async Task<Stream> GetFileFrameAsync(long version, FrameRange range, Canc
         return stream;
     }
 
+    /// <inheritdoc />
+    public async Task<BinaryData> GetFileInRangeAsync(long version, FrameRange range, CancellationToken cancellationToken)
```

Comment:
Where is this used?

Reply:
This is used UpdateInstanceService to get first block content